professor2-toolfile.spec

```diff
@@ -0,0 +1,22 @@
+### RPM external professor2-toolfile 1.0
+Requires: professor2
+%prep
+
+%build
+
+%install
+
+mkdir -p %i/etc/scram.d
+cat << \EOF_TOOLFILE >%i/etc/scram.d/professor2.xml
+<tool name="professor2" version="@TOOL_VERSION@">
+<client>
+<environment name="PROFESSOR2_BASE" default="@TOOL_ROOT@"/>
+</client>
+<runtime name="PATH" value="$PROFESSOR2_BASE/bin" type="path"/>
+<runtime name="PYTHONPATH" value="$PROFESSOR2_BASE/lib/python@PYTHONV@/site-packages" type="path"/>
```

Comment:
You are missing these dependencies listed here: py2-numpy py2-sympy py2-matplotlib root cython yoda eigen